src/style.css

```diff
@@ -59,9 +59,11 @@
   --color-primary: #ff0000;
   --color-text: #2e353f;
   --color-text-light: #4f5969;
+  --color-light-blue: #3352dd;
```

Comment:
Looks like existing Gatsby colors have a name patter like `--color-{NAME}-{VARIANT}`. Maybe consider following it too?

```diff
- --color-light-blue:
+ --color-blue-light:
```